server/base/src/main/java/org/apache/accumulo/server/fs/VolumeChooserEnvironment.java

```diff
@@ -35,17 +38,37 @@
   private final ServerContext context;
   private final ChooserScope scope;
   private final Table.ID tableId;
+  private final TabletId tabletId;
 
   public VolumeChooserEnvironment(ChooserScope scope, ServerContext context) {
     this.context = context;
     this.scope = Objects.requireNonNull(scope);
     this.tableId = null;
+    tabletId = null;
+  }
+
+  @VisibleForTesting
+  public VolumeChooserEnvironment(Table.ID tableId) {
+    this(tableId, null, null);
   }
 
   public VolumeChooserEnvironment(Table.ID tableId, ServerContext context) {
+    this.scope = ChooserScope.TABLE;
+    this.context = context;
+    this.tableId = Objects.requireNonNull(tableId);
```

Comment:
This implementation could call `this(tableId, null, context)`

Reply:
Fixed in next commit